### PR TITLE
Update ProductPrice README

### PR DESCRIPTION
**Pull request by GustavoMA**
#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +21,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Fixed
 - Fix `Logo` to only use maxWidth and maxHeight to define it's size.
 
+### Added
```
Comment (GustavoMA) [inline CHANGELOG.md]: This changelog was lost in a rebase. Just added here to fix that.

File: react/components/ProductPrice/README.md [hunk 2]
```diff
@@ -1,70 +1,65 @@
 # Product Price
```
Comment (paul0vinicius) [inline react/components/ProductPrice/README.md]: I think it's best to add `VTEX` before the title, like `VTEX Product Price`.

File: react/components/ProductPrice/README.md [hunk 3]
```diff
@@ -1,70 +1,71 @@
...
-import ProductPrice from 'vtex.store-components/ProductPrice'
-```
+:loudspeaker: **Disclaimer:** Don't fork this project, use, contribute, or open issue with your feature request.
+
+## Table of Contents
+- [Usage](#usage)
+  - [Blocks API](#blocks-api)
+    - [Configuration](#configuration)
+  - [Styles API](#styles-api)
+    - [CSS namespaces](#css-namespaces)
```
Comment (GustavoMA) [inline react/components/ProductPrice/README.md]:
```suggestion
    - [CSS Namespaces](#css-namespaces)
```

File: react/components/ProductPrice/README.md [hunk 4]
```diff
@@ -1,70 +1,71 @@
 # Product Price
 
-Product Price is a canonical component that any VTEX app can import.
+## Description
 
-To import it into your code:
+`ProductPrice` is a VTEX component that shows the price of a product.
+This Component can be imported and used by any VTEX app.
```
Comment (paul0vinicius) [inline react/components/ProductPrice/README.md]: `This component`

put component in lowercase, just a suggestion.

Comment (GustavoMA) [inline react/components/ProductPrice/README.md]:
```suggestion
This component can be imported and used by any VTEX app.
```

Comment (GustavoMA) [inline react/components/ProductPrice/README.md]:
```suggestion
This component can be imported and used by any VTEX app.
```

File: react/components/ProductPrice/README.md [hunk 5]
```diff
@@ -1,70 +1,71 @@
...
-| `installmentClass` | `String` | Classes to be applied to installment element |
-| `installmentContainerClass` | `String` | Classes to be applied to installment container |
-| `interestRateClass` | `String` | Classes to be applied to interest rate element |
-
-See an example at [Product Summary](https://github.com/vtex-apps/product-summary) app
+For now this block does not have any required or optional blocks.
+
+#### Configuration
+
+Through the Storefront, you can change the `ProductPrice`'s behavior and interface. However, you also can make in your theme app, as Store theme does. However, you also can make in your theme app, as Store theme does.
```
Comment (paul0vinicius) [inline react/components/ProductPrice/README.md]:
```suggestion
Through the Storefront, you can change the `ProductPrice`'s behavior and interface. However, you also can make in your theme app, as Store theme does.
```

This last sentence is duplicated.

**Pull request merged**